lib/internal/Magento/Framework/GraphQl/Query/Resolver/Argument/FieldEntityAttributes.php

```diff
@@ -55,12 +64,12 @@ public function getEntityAttributes() : array
             $configElement = $this->config->getConfigElement($interface['interface']);
 
             foreach ($configElement->getFields() as $field) {
-                $fields[$field->getName()] = 'String';
+                $fields[$field->getName()] = '';
```

Comment:
Why this was changed? I believe each filter value must be of a `String` type. Same below.